ci/moose_utils.py

```diff
@@ -42,10 +42,7 @@ def _download_moose_file(opsys: str, arch: str, file_name: str):
     if nexus_url is None:
         raise ValueError("LIBTELIO_ENV_SEC_NEXUS_URL not set")
 
-    if opsys == "qnap":
-        url = f"{nexus_url}/repository/ll-gitlab-release/{MOOSE_PROJECT_ID}/{LIBTELIO_ENV_MOOSE_RELEASE_TAG}/bin/common/linux/{arch}/{file_name}"
-    else:
-        url = f"{nexus_url}/repository/ll-gitlab-release/{MOOSE_PROJECT_ID}/{LIBTELIO_ENV_MOOSE_RELEASE_TAG}/bin/common/{opsys}/{arch}/{file_name}"
+    url = f"{nexus_url}/repository/ll-gitlab-release/{MOOSE_PROJECT_ID}/{LIBTELIO_ENV_MOOSE_RELEASE_TAG}/bin/common/{opsys}/{arch}/{file_name}"
```

Comment:
`opsys` stands for OS I suppose, maybe worth renaming?

Reply:
Yeah, a little confusing, renamed